bazel/deps.bzl

```diff
@@ -23,6 +23,8 @@ BF_SDE_PI_VER = {
     "9_1_0": "41358da0ff32c94fa13179b9cee0ab597c9ccbcc",
     "9_2_0": "4546038f5770e84dc0d2bba90f1ee7811c9955df",
     "9_3_0": "4546038f5770e84dc0d2bba90f1ee7811c9955df",
+    "9_3_1": "4546038f5770e84dc0d2bba90f1ee7811c9955df",
+    "9_4_0": "4546038f5770e84dc0d2bba90f1ee7811c9955df",
```

Comment:
@Yi-Tseng to confirm the PI versions.

Reply:
yes, `4546038f5770e84dc0d2bba90f1ee7811c9955df` is the correct one.

Reply:
Thanks for confirming.